librz/analysis/type_pdb.c

```diff
@@ -28,237 +49,634 @@ static char *create_type_name_from_offset(ut64 offset) {
 	return str;
 }
 
+static RzType *parse_type_array(const RzTypeDB *typedb, STypeInfo *type) {
+	SLF_ARRAY *lf_array = type->type_info;
+	RzType *typ = RZ_NEW0(RzType);
+	if (!typ) {
+		return NULL;
+	}
+	typ->kind = RZ_TYPE_KIND_ARRAY;
+	SType *element = get_stype_by_index(lf_array->element_type);
+	if (!element) {
+		rz_type_free(typ);
+		return NULL;
+	}
+	RzType *element_type = parse_type(typedb, element);
+	if (!element_type) {
+		rz_type_free(typ);
+		return NULL;
+	}
+	typ->array.type = element_type;
+	typ->array.count = type->get_val(type);
+	return typ;
+}
+
+static RzType *parse_regular_type(const RzTypeDB *typedb, SType *type) {
+	STypeInfo *type_info = &type->type_data;
+	switch (type_info->leaf_type) {
+	case eLF_CLASS:
+	case eLF_CLASS_19:
+		// todo
```

Comment:
Could you please put a link to the issue here as well? https://github.com/rizinorg/rizin/issues/1205